FreestarOBJCSample/Podfile

```diff
@@ -1,23 +1,45 @@
+platform :ios, '11.0'
+use_frameworks!
 
 target 'FreestarOBJCSample' do
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
-  pod 'FreestarAds-PreRelease', '5.2.0-beta-1'
-  pod 'FreestarAds-AdColony-PreRelease', '~> 5.0-alpha'
-  pod 'FreestarAds-AppLovin-PreRelease', '~> 5.1.1-beta-1'
-  pod 'FreestarAds-Mopub-PreRelease', '5.1.0-beta-1'
-  pod 'FreestarAds-Googleadmob-PreRelease', '~> 6.1.1-beta-1'
-  pod 'FreestarAds-Tapjoy-PreRelease', '~> 5.0-alpha'
-  pod 'FreestarAds-Unity-PreRelease', '~> 6.0-alpha'
-  pod 'FreestarAds-Vungle-PreRelease', '~> 5.1.1-beta-1'
-#   pod 'FreestarAds-Google-PreRelease', '~> 4.0-alpha' #incompatible with latest nimbus
-  pod 'FreestarAds-Criteo-PreRelease', '~> 3.0-alpha'
-  pod 'FreestarAds-GAM-PreRelease', '~> 5.1.1-beta-1'
-  pod 'FreestarAds-Nimbus-PreRelease', '~> 3.1.1-beta-1'
-  pod 'FreestarAds-TAM-PreRelease', '~> 2.1.1-beta-1'
-  pod 'FreestarAds-Pangle-PreRelease', '~> 1.1.1-beta-1'
-  pod 'FreestarAds-Hyprmx-PreRelease', '1.0.0-beta-1'
+  # pod 'FreestarAds-PreRelease', '5.6.0-beta-1'
+  # pod 'FreestarAds-AdColony-PreRelease', '~> 5.3.0-alpha-1'
+  # pod 'FreestarAds-AppLovin-PreRelease', '~> 5.3.0-alpha-1'
+  # pod 'FreestarAds-Mopub-PreRelease', '~> 5.3.0-alpha-1'
+  # pod 'FreestarAds-Googleadmob-PreRelease', '~> 6.3.0-beta-2'
+  # pod 'FreestarAds-Tapjoy-PreRelease', '~> 5.2.0-alpha-1'
+  # pod 'FreestarAds-Unity-PreRelease', '~> 6.2.0-alpha-1'
+  # pod 'FreestarAds-Vungle-PreRelease', '~> 5.4.0-alpha-1'
+  # pod 'FreestarAds-Criteo-PreRelease', '~> 3.2.0-alpha-1'
+  # pod 'FreestarAds-GAM-PreRelease', '~> 5.3.0-beta-2'
+  # pod 'FreestarAds-Nimbus-PreRelease', '~> 3.4.0-beta-3'
+  # pod 'FreestarAds-TAM-PreRelease', '~> 2.4.0-beta-1'
+  # pod 'FreestarAds-Pangle-PreRelease', '~> 1.3.0-alpha-1'
+  # pod 'FreestarAds-Hyprmx-PreRelease', '~> 1.2.0-alpha-1'
+  # pod 'FreestarAds-Google-PreRelease', '~> 5.1.0-beta-1'
+
+  pod 'FreestarAds-PreRelease', '~> 5.15.0-beta-3'
+  pod 'FreestarAds-Ogury-PreRelease', '~> 2.1.0.3-beta-5'
+  #  pod 'FreestarAds-Prebid-PreRelease', '~> 2.0.0.3-beta-3'
+  #  pod 'FreestarAds-Yahoo-PreRelease', '~> 1.14.3.2-beta-7' 
+  #  pod 'FreestarAds-AppLovinMax-PreRelease', '~> 11.0.0.7-beta-4'
+  #  pod 'FreestarAds-AppLovin-PreRelease', '~> 11.0.0.1-beta'
+  #  pod 'FreestarAds-AdColony-PreRelease', '~> 4.7.2.1-beta'
+  #  pod 'FreestarAds-Tapjoy-PreRelease', '~> 12.8.0.1-beta'
+  #  pod 'FreestarAds-Criteo-PreRelease', '~> 4.3.1.1-beta'
+  #  pod 'FreestarAds-TAM-PreRelease', '~> 4.4.3.2-beta-3'
+  #  pod 'FreestarAds-Hyprmx-PreRelease', '~> 6.0.0.0-beta'
+  #  pod 'FreestarAds-Hyprmx-PreRelease/MRAID', '~> 6.0.0.0-beta'
+  #  pod 'FreestarAds-Googleadmob-PreRelease', '~> 9.2-beta'
+  #  pod 'FreestarAds-Googleadmob-PreRelease/Facebook', '~> 9.2-beta'
+  #  pod 'FreestarAds-GAM-PreRelease', '~> 9.2-beta'
+  #  pod 'FreestarAds-GAM-PreRelease/Facebook', '~> 9.2-beta'
+  #  pod 'FreestarAds-Unity-PreRelease', '~> 4.1.0.4-beta-1'
 
```

Comment:
@carlosalcala-freestar i don't think anurag wants you to remove or uncomment all the other pods from the Podfile... @anurag-sdk-apps